guides/plugins/plugins/plugin-fundamentals/programmically-sending-emails.md

```diff
@@ -0,0 +1,119 @@
+# Programmically sending Emails
+
+In this guide you'll learn how to send Emails programmically using the `Mailservice` provided in Shopware v6.4.x.
```

Comment:
lowercase "emails".
`MailService` (camel-case)
typo - "programmatically"

> In this guide you'll learn how to send Emails programmically using the `Mailservice`.

And then use a {% hint type="info" %} to refer to the version constraint:

```
{% hint style="info" %}
This guide only applies to Shopware versions greater than 6.4.0.
{% endhint %}
```

Reply:
Done, thanks :+1: